modules/presentation.md

```diff
@@ -1,4 +1,4 @@
-# Presentation Hints
+# Divina Presentation Hints
```

Comment:
Should this be moved under `profiles/divina.md` instead of being a module?

Reply:
Good question, there's very little left in this document, it could make sense.

There's a similar situation with transitions: they currently live in their own doc instead of the Divina profile doc.

Reply:
I'm going to update this PR and move the remaining presentation properties to the Divina spec. I'll leave the doc for transitions aside for now.

Reply:
Looking back at what remains, I don't think it should go in the Divina doc at all. All of the remaining properties are too experimental in their potential use case IMO. I know that @llemeurfr might disagree, but I think that given all the current on-going discussions, we would benefit from a streamlined take on Divina.

I would move the remaining content back into a draft PR and we'll see if it makes sense to re-introduce them back at some point.